packages/web/src/js/main.js

```diff
@@ -0,0 +1,73 @@
+const StringifyData = new URLSearchParams(window.location.search).get('Data');
+const Data = JSON.parse(StringifyData);
+
+const DateTextElement = document.getElementById('Date-Text');
+
+const WeekCounter = (DateString) => {
```

Comment:
we use camelcase for naming function in JS.
You should get eslint warning for that, I don't know whats wrong with our eslint.